mdnDocsOverrides.json

```
@@ -273,8 +283,12 @@
   "bigint": [
     "Web/JavaScript/Reference/Global_Objects/BigInt"
   ],
-  "bigint64array": [],
-  "blocking-render": [],
+  "bigint64array": [
+    "Web/JavaScript/Reference/Global_Objects/BigInt64Array"
+  ],
+  "blocking-render": [
+    "Web/API/HTMLStyleElement/blocking"
+  ],
```

This feature is also for `<link>` and `<script>`. Let's add them for completeness:

```suggestion
  "blocking-render": [
    "Web/API/HTMLLinkElement/blocking"
    "Web/API/HTMLScriptElement/blocking"
    "Web/API/HTMLStyleElement/blocking"
  ],
```